cli/README.md

```diff
@@ -0,0 +1,42 @@
+# generator-mern [![NPM version][npm-image]][npm-url] [![Dependency Status][daviddm-image]][daviddm-url]
+> A Yeoman generator to scaffold a MERN app
+
+## Installation
+
+First, install [Yeoman](http://yeoman.io) using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)).
+
+```bash
+npm install -g yo
+```
+
+Then link the generator-mern because is not available as global npm module. On the command line, from the root of your generator project (in the /cli folder), type:
+
+```bash
+npm link
+```
+
+That will install your project dependencies and symlink a global module to your local file.
+
+## Generators
+
+### Component
+
+Generates a component in `app/components`.
+
+Example:
+
+```bash
+yo mern:component Sidebar
```

Comment:
👍 @matiasbeckerle thanks! 
No problem, I can explain the insights of each one when you want! thanks 💯